### DISCUSSION TITLE
ANTV

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#354] [Show and tell]
### Channel ID

ANTV.cv

### Stream URL

https://vod.livestream.com/630e6da3_4f1a40ac128429af563cf957eaeb8a83ab684e5f/events/0000000000a11a31/4a721cc5-ef2b-4d1e-a303-595e7522ab43_1756-19.ts

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
https://livestream.com/accounts/19284389/events/10558001  not 24/7

**Comment by freearhey** [comment]
https://vod.livestream.com/630e6da3_4f1a40ac128429af563cf957eaeb8a83ab684e5f/events/0000000000a11a31/4a721cc5-ef2b-4d1e-a303-595e7522ab43_1756-19.ts (offline)
<img width="830" alt="image" src="https://user-images.githubusercontent.com/7253922/232234738-71e31276-3048-4b21-aee6-98d5715599ab.png">
